tests/Unit/BlamableServiceProviderTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Astalpaert\LaravelBlamable\Tests\Unit;
+
+use Astalpaert\LaravelBlamable\Tests\TestCase;
+use Illuminate\Database\Schema\Blueprint;
+
+class BlamableServiceProviderTest extends TestCase
+{
+    /** @test **/
+    public function it_has_registers_macros()
```

Comment:
I think it's either `it_has_macros` or `it_registers_macros`

Reply:
Good point, renamed to it_registers_macros